mautic/mautic
# Fix import "Don't overwrite value if already exists" for company

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:
If you import company  and you enable **Don't overwrite value if already exists**  company was overwritten even If you do not expect it.  This PR fixed it 



1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create contact with company Mautic
3. Create import file for this contact with company **Mautic2**. CSV could look like
```csv
email,company
test@test.com,Mautic2
```
4. Contact should assign to company Mautic2 after import
5. Then import file for contact with company Mautic3 but enabled **Don't overwrite value if already exists**. CSV could look like 
```csv
email,company
test@test.com,Mautic3
```
6. Contact should not assign to company Mautic3 after import






<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10719"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
